userland/examples/ble-serialization/nrf51_serialization.c

```diff
@@ -0,0 +1,20 @@
+#include "nrf51_serialization.h"
```

Comment:
Any reason these aren't included in `libtock`? A signpost radio app uses nrf-serialization, so it would have to copy these two files into itself as well if they aren't in `libtock` anymore.